python-nectar.spec

```diff
@@ -1,9 +1,11 @@
 %{!?python_sitelib: %global python_sitelib %(%{__python} -c "from %distutils.sysconfig import get_python_lib; print get_python_lib()")}
+%{!?python3_sitelib: %global python3_sitelib %(%{__python3} -c "from %distutils.sysconfig import get_python_lib; print(get_python_lib())")}
+%global sum A download library that separates workflow from implementation details
```

Comment:
Maybe you can drop this global definition, put text to Summary: and next use %{summary} macro.